### ISSUE TITLE
request:8bitdo usb switch adapter

### ISSUE THREAD

**Issue by diamondanimates** [FIX94/Nintendont#856]
can you add this? i got this for christmas, but dont want to buy a new mayflash adapter.

**Comment by avLightless** [comment]
I have tried the 8bitdo adapter with all the different input modes (xinput, dinput, mac) but none of them work with the default controller configs. I dont know if it would work with a custom config though. One thing I have noticed, is that when switching to the mac input mode (I believe its holding select + right dpad for 3 seconds) my game received 5 or 6 UP inputs in the menu. So there is definitely some compatibility there.

**Comment by carnage702** [comment]
all controller.inis that exist were made by regular people and if you have a device that isnt coded you must do your own controller.ini this is for issues and not for requesting inis.

**Comment by gdezso** [comment]
"...my game received 5 or 6 UP inputs in the menu. So there is definitely some compatibility there."

I found, what caused that. The 'RightUp', 'DownRight', 'DownLeft' and 'UpLeft' using the wrong offset, which is 2. And the offset 2 should belong to 'StickY' only. If you change the offset from 2 to 5 for the mentioned DPad directions, the issue will be solved. Tested on Nintendont latest build with SN30PRO firmware v1.35
